backend/src/main/java/de/otto/platform/gitactionboard/config/security/WebSecurityConfig.java

```diff
@@ -54,11 +54,13 @@ private static HttpSecurity getDefaultSettings(HttpSecurity http) throws Excepti
   @Bean
   public List<AuthenticationMechanism> availableAuths(
       @Value("${BASIC_AUTH_USER_DETAILS_FILE_PATH:}") String basicAuthDetailsFilePath,
+      @Value("${BASIC_AUTH_USER_DETAILS:}") String basicAuthDetailsContent,
```

Comment:
Given that we have multiple of ways to configure basic auth users, it's better to get those list of users as argument and check the **size** of the list to validate whether basic auth is enabled or not